portal-impl/src/com/liferay/portal/security/permission/AdvancedPermissionChecker.java

```diff
@@ -764,6 +764,12 @@ else if ((groupId > 0) &&
 			newIndividualResourcePrimKey = name;
 		}
 
+		else if (primKey.equals(
+					String.valueOf(ResourceConstants.PRIMKEY_DNE))) {
+
+			newIndividualResourcePrimKey = name;
+		}
+
```

Comment:
Hey Tomas,

Thanks for looking into it. I'll send you the pull so that you can make the changes.